sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -25,6 +25,7 @@ import { LoggingPolicyFactory } from "./LoggingPolicyFactory";
 import { RetryOptions, RetryPolicyFactory } from "./RetryPolicyFactory";
 import { TelemetryOptions, TelemetryPolicyFactory } from "./TelemetryPolicyFactory";
 import { UniqueRequestIDPolicyFactory } from "./UniqueRequestIDPolicyFactory";
+import { isBrowser } from "../test/utils";
```

Comment:
There's usage of `isNode` in the code already. I think it's better than importing from test utils.